buildSrc/src/main/kotlin/plugin/AppConfigurer.kt

```diff
@@ -50,9 +50,7 @@ class AppConfigurer {
           )
         }
       }
-      dexOptions {
-        javaMaxHeapSize = "4g"
-      }
+
```

Comment:
Why should we change this now?

Reply:
Hi @gouri-panda ,
in gradle plugin 8.0 dexing automatically optimized.
Full warning is
```
WARNING:DSL element 'dexOptions' is obsolete and should be removed.
It will be removed in version 8.0 of the Android Gradle plugin.
Using it has no effect, and the AndroidGradle plugin optimizes dexing automatically.
```